sycl/test/on-device/group_algorithms_sycl2020/all_of.cpp

```diff
@@ -0,0 +1,65 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -I . -o %t.out
+// TODO: re-enable HOST execution line when this test is moved to
```

Comment:
As for my understanding group algorithms are not supported on HOST. Do we really need this comment?

Reply:
Good question. The original ONEAPI tests these are based upon all had a HOST execution line.  I only disabled it because the way one invokes the HOST on the intel/llvm test is not the same as llvm-test-suite.  I agree that I think we should just remove it.